README.md

```diff
@@ -554,7 +554,7 @@ To see what has changed in recent versions, see the [CHANGELOG](CHANGELOG.md).
 
 ## Known Limitations
 
-This library currently does not support being the [target of a push notification](https://www.ably.com/documentation/general/push#activate) (i.e. web push).
+This library currently does not support being the [target of a push notification](https://www.ably.com/docs/general/push#activate) (i.e. web push).
```

Comment:
The activate fragment doesn't seem to exist on that page, there is activate-device but maybe this should link to https://ably.com/docs/general/push#platform-support instead?

Reply:
[8864f57](https://github.com/ably/ably-js/pull/964/commits/8864f579cc858fb7207cae17afa45735c55c2454)